R/servers.R

```diff
@@ -88,12 +95,12 @@ isConnectServer <- function(server) {
   !isCloudServer(server) && !isRPubs(server)
 }
 
-cloudServerInfo <- function(name) {
+cloudServerInfo <- function(name, url) {
   name <- arg_match0(name, cloudServers)
```

Comment:
I think you could drop this line, and then it would be possible to inline `positCloudServers` into `isPositCloudServer()` and delete `cloudServers`.